src/config/delegates.json

```diff
@@ -1,4 +1,16 @@
 {
```

Comment:
Please don't use upper-case chars in file names as it can confuse git in certain cases. Rename the image file to `refi-col.jpg`

Reply:
Done